spec/tanker/identity_spec.rb

```diff
@@ -286,7 +286,7 @@ def assert_signature(identity, app_public_key)
       public_identity = Tanker::Identity.deserialize(b64_public_identity)
       expect(public_identity.keys.sort).to eq ['public_encryption_key', 'public_signature_key', 'target', 'trustchain_id', 'value']
       expect(public_identity['trustchain_id']).to eq @app[:id]
-      expect(public_identity['target']).to eq 'email'
+      expect(public_identity['target']).to eq 'hashed_email'
```

Comment:
I expected the related fix in `lib/tanker/identity.rb` to be in this first commit 🤔

Reply:
Indeed, thank you. I forgot the rebase would not carry that over.